Tests/ClientTests/CreditCard/CreditCardValidatorTests.swift

```diff
@@ -0,0 +1,118 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+@testable import Client
+
+class CreditCardValidatorTests: XCTestCase {
```

Comment:
That's great progress but I run the test and there are some cases that are not being tested yet for instance the 
![Screenshot 2023-04-03 at 10 24 35 AM](https://user-images.githubusercontent.com/18538058/229539379-7477cb9a-5852-481b-9c23-8d0a0d637f4f.png)
Actually to be really thorough there should be 1 valid test per card type (visa, mastercard, etc)